_pages/resources/training-material.md

```diff
@@ -7,4 +7,31 @@ sidebar:
   nav: "uhpc_menu"
 ---
 
-We will upload our training material very soon.
+## Course Material
```

Comment:
I wonder if a bit more context at the top of the page might be useful? At the moment it feels like this section jumps straight into details of how the material is developed without an overview of what's actually here - if this is primarily aimed at a developer/trainer audience, I guess that's fine but I'm also assuming we may get individuals keen to learn from the materials accessing this page. I'd therefore suggest some intro text along the lines of the following:

```suggestion
This section of the UNIVERSE-HPC website provides access to our collated
training materials and learning pathway structures. The materials here are
targeted at both trainers / course developers and individuals who are keen to
develop their skills through self-learning. Below you can find a table of our
current [courses and modules](#courses-and-modules). You can also find details
aimed at training material developers and course organisers about how the
material is built and how to deploy your own version of the reference course
website.

Pathway structures can be viewed via a series of interactive graphs accessible
via the course links in the table below and via the course material section of
the "Gutenberg" [course website](#course-website).

## Course Material
```

What do you think?